scripts/deploy_gce_ci.sh

```diff
@@ -4,37 +4,39 @@
 cd $GOPATH/src/github.com/google/trillian
-docker build --quiet -f examples/deployment/docker/log_server/Dockerfile -t gcr.io/${PROJECT_NAME_CI}/log_server:${TRAVIS_COMMIT} .
-docker build --quiet -f examples/deployment/docker/log_signer/Dockerfile -t gcr.io/${PROJECT_NAME_CI}/log_signer:${TRAVIS_COMMIT} .
+docker build --quiet -f examples/deployment/docker/log_server/Dockerfile -t gcr.io/${PROJECT_NAME}/log_server:${TRAVIS_COMMIT} .
+docker build --quiet -f examples/deployment/docker/log_signer/Dockerfile -t gcr.io/${PROJECT_NAME}/log_signer:${TRAVIS_COMMIT} .
 
 echo "Pushing docker images..."
-gcloud docker -- push gcr.io/${PROJECT_NAME_CI}/log_server:${TRAVIS_COMMIT}
-gcloud docker -- push gcr.io/${PROJECT_NAME_CI}/log_signer:${TRAVIS_COMMIT}
+gcloud docker -- push gcr.io/${PROJECT_NAME}/log_server:${TRAVIS_COMMIT}
+gcloud docker -- push gcr.io/${PROJECT_NAME}/log_signer:${TRAVIS_COMMIT}
 
 echo "Tagging docker images..."
-gcloud --quiet container images add-tag gcr.io/${PROJECT_NAME_CI}/log_server:${TRAVIS_COMMIT} gcr.io/${PROJECT_NAME_CI}/log_server:latest
-gcloud --quiet container images add-tag gcr.io/${PROJECT_NAME_CI}/log_signer:${TRAVIS_COMMIT} gcr.io/${PROJECT_NAME_CI}/log_signer:latest
+gcloud --quiet container images add-tag gcr.io/${PROJECT_NAME}/log_server:${TRAVIS_COMMIT} gcr.io/${PROJECT_NAME}/log_server:latest
+gcloud --quiet container images add-tag gcr.io/${PROJECT_NAME}/log_signer:${TRAVIS_COMMIT} gcr.io/${PROJECT_NAME}/log_signer:latest
 
 echo "Updating jobs..."
 kubectl delete configmap deploy-config
-kubectl create -f examples/deployment/kubernetes/trillian-opensource-ci.yaml
-
-kubectl apply -f examples/deployment/kubernetes/trillian-log-deployment.yaml
-kubectl apply -f examples/deployment/kubernetes/trillian-log-service.yaml
-kubectl apply -f examples/deployment/kubernetes/trillian-log-signer-deployment.yaml
-kubectl apply -f examples/deployment/kubernetes/trillian-log-signer-service.yaml
-kubectl set image deployment/trillian-logserver-deployment trillian-logserver=gcr.io/${PROJECT_NAME_CI}/log_server:${TRAVIS_COMMIT}
-kubectl set image deployment/trillian-logsigner-deployment trillian-log-signer=gcr.io/${PROJECT_NAME_CI}/log_signer:${TRAVIS_COMMIT}
+envsubst < ${DIR}/../examples/deployment/kubernetes/${CONFIGMAP} | kubectl create -f -
+
+envsubst < ${DIR}/../examples/deployment/kubernetes/trillian-log-deployment.yaml | kubectl apply -f -
+envsubst < ${DIR}/../examples/deployment/kubernetes/trillian-log-service.yaml | kubectl apply -f -
+envsubst < ${DIR}/../examples/deployment/kubernetes/trillian-log-signer-deployment.yaml | kubectl apply -f -
+envsubst < ${DIR}/../examples/deployment/kubernetes/trillian-log-signer-service.yaml | kubectl apply -f -
+kubectl set image deployment/trillian-logserver-deployment trillian-logserver=gcr.io/${PROJECT_NAME}/log_server:${TRAVIS_COMMIT}
+kubectl set image deployment/trillian-logsigner-deployment trillian-log-signer=gcr.io/${PROJECT_NAME}/log_signer:${TRAVIS_COMMIT}
```

Comment:
Would it possible/helpful to commonize parts of this with `deploy.sh`?

Reply:
possibly, though maybe for another PR?